unlock-app/src/hooks/usePaywallLocks.ts

```diff
@@ -17,13 +17,13 @@ export const usePaywallLocks = (lockAddresses: string[]) => {
     })
 
     const locks = await Promise.all(lockPromises)
-    setLocks(locks)
     setLoading(false)
+    setLocks(locks)
```

Comment:
We now stop loading before we set the locks, this way we don't show both the placeholder locks and the real locks at the same time.